Episode 17.1 - Illusion (#6643)
rathena/rathena: npc/re/quests/quests_17_1.txt

		mes "There will be Rekenber guards sent by Aas and troop dispatched by the republic.";
		next;
		mes "[Tes]";
		mes "Est will also be there, you'll find the location easily.";
		mes "I hope this mission goes well.";
		close3;
	}
	if (isbegin_quest(7862) == 2 && isbegin_quest(7864) == 0) {
		cutin "ep16_tes01",2;
		mes "[Tes]";
		mes "You're back.";
		mes "Est came back first and I wasn't sure something had happened to you.";
		next;
		mes "[Tes]";
		mes "The information I got from Est was incredible.";
		mes "And it seems Lazy has something to tell you.";
		next;
		mes "[Tes]";
		mes "Kaya should be there too.";
		mes "I have to analyze the data everyone is giving me, it's piling up.";
		mes "You have enough to do already.";
		setquest 7864;
		getitem 25669,25;
		getexp 800000,800000;
		cloakoffnpc "Lazy#171_cat",getcharid(0);
		cloakoffnpc "Kaya Tos#171_cat",getcharid(0);
		cloakoffnpc "Aas#171_cat",getcharid(0);
		close3;
	}
	if (isbegin_quest(7863) == 1) {
		cutin "ep16_tes01",2;
		mes "[Tes]";
		mes "Is it related to the laboratory's security.";
		mes  "Aas told me you had something to deliver.";
		next;
		mes "[Tes]";

> 		mes "I think it's a concious decision from Schlein to have left these documents.";

Reviewer: ```suggestion
		mes "I think it's a conscious decision from Schlein to have left these documents.";
```

		close3;
	}
	if (isbegin_quest(7864) == 1) {
		cutin "ep16_tes01",2;
		mes "[Tes]";
		mes "Lazy seems to be up to something.";
		mes "He's somewhere here in the pub, go and find him.";
		cloakoffnpc "Lazy#171_cat",getcharid(0);
		cloakoffnpc "Kaya Tos#171_cat",getcharid(0);
		cloakoffnpc "Aas#171_cat",getcharid(0);
		close3;
	}
	if (isbegin_quest(17014) == 1) {
		cutin "ep16_tes01",1;
		mes "[Tes]";
		mes "I thought you were investigating with Lazy, it looks like you didn't get great results.";
		next;
		mes "[Tes]";
		mes "I finished analyzing the information you gathered from Regenschirm earlier, there were traces of supplies and research data.";
		next;
		mes "[Tes]";
		mes "So far, I believe it was a trick from the Enterprise, from those who sit back at the company.";
		mes "Huh...";
		cloakoffnpc "Kaya#171_cat3",getcharid(0);
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";
		mes "I agree with your assessment.";
		mes "The more I learn about the truth, the less I understand what's going on.";
		npctalk "Oh, what happened?","",bc_self;
		next;
		mes "[Kaya]";
		mes "What I've seen so far was only the tip of the iceberg.";
		mes "My eyes are now wide opened.";

> 		npctalk "I'm sorry to interupt so suddenly.","Kaya#pub_cat_dummy",bc_self;

Reviewer: ```suggestion
		npctalk "I'm sorry to interrupt so suddenly.","Kaya#pub_cat_dummy",bc_self;
```

		mes "[Tes]";
		mes "That's not it, Est.";
		mes "You do know there's a reason for that, right?";
		next;
		cutin "ep162_est03",2;
		mes "[Est]";
		mes "I've already decided to send a support unit.";
		mes "Even so, is that really going to be enough?";
		npctalk ".....","Middle-Aged Man#171_cat",bc_self;
		npctalk "....hick...","Rekenber Senior Guard#171_cat",bc_self;
		next;
		select("So what's going on?");
		mes "[Est]";
		mes "I'm a little too excited. I'll shut up. Tes, explain it please.";
		next;
		cutin "ep16_tes01",0;
		mes "[Tes]";
		mes "To put it simply, our affiliated Enterprise research institute, Regenschirm, is occupied by a large group of enemies.";
		mes "A special unit under Est's command will be dispatched for the recapture.";
		next;
		mes "[Tes]";
		mes "So that's why Est was so nervous.";
		next;
		select("Why aren't the Enterprise guards helping?");
		mes "[Tes]";
		mes "Well you see...";
		npctalk "That's all my fault!","Rekenber Senior Guard#171_cat",bc_self;
		next;
		cutin "ep171_as01",1;
		mes "[Aas]";
		mes "It all started because of the traitor within the guards.";
		mes "Oh, sorry, I'm senior Rekenber security officer, Aas.";
		next;
		mes "[Aas]";

> 		mes "The traitor is the chief security officer, Schlein, who has substancial support within the guard forces.";

Reviewer: ```suggestion
		mes "The traitor is the chief security officer, Schlein, who has substantial support within the guard forces.";
```

		next;
		cutin "ep171_kaya03",2;
		mes "[Kaya]";
		mes "Isn't it common sense that ^0000CDpeople inside the Enterprise know more?^000000";
		mes "Not only would we have access to more data, but the quality would be much higher than what's available to an outsider.";
		next;
		cutin "16lei_02",0;
		mes "[Lazy]";
		mes "Don't you think the Vice President would only share with us ^0000CDwhat he want us to know?^000000";
		mes "Am I supposed to suspend all critical thinking and unquestioningly believe you?";
		next;
		mes "[Lazy]";
		mes "Won't that mean you can just report to the President and hide a plan from us? Yeah? Right?";
		npctalk "No big crowds right?","",bc_self;
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";
		mes "I never said that.";
		next;
		select("Lazy, what's wrong?");
		cutin "16lei_01",0;
		mes "[Lazy]";
		mes "Adventurer? Adventurer!! Hey, I haven't died yet!! Never been better!!";
		mes "Listen to me.";
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";
		mes "Lazy? Is your name Lazy? Didn't you introduce yourself as Daisy?";
		next;
		cutin "16lei_02",0;
		mes "[Lazy]";

> 		mes "Oh, that? Don't worry about it, it's the name I use to distract people. There, now you know me. Oh by the way... nevermind, you don't need to know that.";

Reviewer: ```suggestion
		mes "Oh, that? Don't worry about it, it's the name I use to distract people. There, now you know me. Oh by the way... never mind, you don't need to know that.";
```

		cutin "16lei_03",0;
		mes "[Lazy]";
		mes "I was born to talk about it, but who are you talking about?";
		mes "I have so many things to talk about. I can talk about them all day long.";
		next;
		mes "[Lazy]";
		mes "Now, as you know, this conversation is business meeting, aren't the Vice President's underlings standing right behind you?";
		npctalk "Who else is getting paid by Rekenber here? The adventurer?","",bc_self;
		npctalk "Are you talking about me?","Aas#171_cat",bc_self;
		next;
		mes "[Lazy]";
		mes "It's already known to our neighbors that Rekenber was performing illegal experiments. Maybe the people on this list could have also been involved.";
		next;
		cutin "16lei_02",0;
		mes "[Lazy]";
		mes "Don't you want to clear their suspicions? Get their cooperation to know what's in your backyard?";
		mes "If it were up to me, I'd have done that already. When else would we get such an opportunity?";
		next;
		cutin "16lei_03",0;
		mes "[Lazy]";
		mes "What more are you hiding in your backyard? Are you not here for asking our help? If there's anything hidden over there,";
		mes "wouldn't the Vice President be in trouble?";
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";

> 		mes "Requesting help and analyzing paperworks are separate matters.";

Reviewer: ```suggestion
		mes "Requesting help and analyzing paperwork are separate matters.";
```

		mes "Payments are also confirmed to be monthly basis.";
		next;
		mes "[Lazy]";
		mes "It's just common sense, why would a company go out their way to pay a salary to an unemployed person?";
		next;
		cutin "16lei_01",2;
		mes "[Lazy]";
		mes "I'm telling you it's common sense~";
		mes "I can say common sense in front of people who have studied hard~";
		next;
		cutin "16lei_02",2;
		mes "[Lazy]";
		mes "Wait, did I misunderstand the meaning of common sense?";
		mes "Did I change the definition without knowing about it?";
		unittalk getcharid(3),strcharinfo(0)+ " : Wouldn't you?",bc_self;
		npctalk "Isn't that what it means?","",bc_self;
		next;
		cutin "16lei_03",2;
		mes "[Lazy]";
		mes "So either you're lying here, or I'm the one lying. Adventurer, am I the one lying?";
		unittalk getcharid(3),strcharinfo(0)+ " : Um...",bc_self;
		npctalk "What are you worried about? I'm a very reliable person, why are you looking at me like that!","",bc_self;
		next;
		cutin "16lei_01",2;
		mes "[Lazy]";
		mes "To sum it all up, we've lost many contacts to Rekenber's employees. Nobody really knows what happened to them.";
		next;
		mes "[Lazy]";
		mes "But they have such a long history?";

> 		mes "The governent just now realized about this issue. There's definitely a connection between that and the missing people. That's why I decided to investigate further.";

Reviewer: ```suggestion
		mes "The government just now realized about this issue. There's definitely a connection between that and the missing people. That's why I decided to investigate further.";
```

		next;
		cutin "16lei_03",2;
		mes "[Lazy]";

> 		mes "Who else would do the job if not me? That's why I have to investage.";

Reviewer: ```suggestion
		mes "Who else would do the job if not me? That's why I have to investigate.";
```

		next;
		cutin "ep171_kaya01",0;
		mes "[Kaya]";
		mes "This is a waste of time, no matter how I look at it. Even if you told me a hundred times, it's not economical for me to go. But I accept your proposal anyway.";
		next;
		cutin "16lei_01",2;
		mes "[Lazy]";
		mes "Economical? Of course it's economical. I also like being economical. So, I can do this economically on my own terms? Yeah?";
		mes "Dear Vice President?";
		npctalk "That settles it, we'll answer to the Vice President. I knew it!","",bc_self;
		next;
		cutin "ep171_kaya01",0;
		mes "[Kaya]";
		mes "Listen to me. I asked the government for help. I do agree that for us to trust each other, we must fully cooperate with one another.";
		next;
		cutin "ep171_kaya02",0;
		mes "[Kaya]";
		mes "If the only way to achieve that is by reviewing the people on the list, I will accept the condition.";
		mes "^0000CDI will accompany you and actively cooperate.^000000";
		next;
		cutin "ep171_as02",1;
		mes "[Aas]";
		mes "Please reconsider, Vice President.";
		mes "You do not have to do that. I should go instead of you, no?";
		next;
		cutin "ep171_kaya02",0;
		mes "[Kaya]";

> 		mes "There is a limit to what you can do. This is my responsiblity and mine alone.";

Reviewer: ```suggestion
		mes "There is a limit to what you can do. This is my responsibility and mine alone.";
```

		next;
		cutin "",255;
		mes "["+ strcharinfo(0) + "]";
		mes "As Lazy said, he doesn't trust your findings.";
		mes "If you can't believe in each other results, why don't you guys start from the beginning?";
		next;
		mes "["+ strcharinfo(0) + "]";
		mes "If you both investigate the differences between the reports, you'll know for sure which is accurate. A compromise must be done here.";
		next;
		cutin "ep171_kaya02",2;
		mes "[Kaya]";
		mes "That's valid argument. But do I need to go too? I would prefer talking to you.";
		next;
		cloakoffnpc "Lazy#171_cat",getcharid(0);
		cloakonnpc "Lazy#171_cat2",getcharid(0);
		cutin "16lei_02",0;
		mes "[Lazy]";
		mes "Can you believe what I've investigated so far? You still can't believe my findings, because you're assuming your documents are correct.";
		next;
		cutin "16lei_03",0;
		mes "[Lazy]";
		mes "Yeah, no. I didn't think so.";
		npctalk "The truth is overwhelming.","Lazy#171_cat",bc_self;
		next;
		select("Do you mind?");
		mes "[Lazy]";
		mes "I can't help it. We're wasting our time with them, why bother? What's wrong? Do you have nothing to say?";
		next;
		select("Please wait a moment, Lazy");
		mes "[Lazy]";

> 		mes "I don't have much time to wait for this sillyness.";

Reviewer: ```suggestion
		mes "I don't have much time to wait for this silliness.";
```

		mes "Huh? I know him! That is indeed Kaya, The Vice President.";
		mes "When I was at the headquarters, I saw him a few times.";
		171_worker_talk(45,.@old$);
		next;
		mes "[Worker]";
		mes "Oh, really? I didn't know because I had never seen your face before.";
		171_worker_talk(46,.@cook$);
		171_worker_talk(47,.@poor$);
		next;
		mes "[Worker]";
		mes "Why is the Vice President here?";
		171_worker_talk(48,.@man$);
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";
		mes "Were you all hired by the Rekenber institute to work on constructing and maintenance in this new lab?";
		next;
		cutin "",255;
		mes "[Worker]";
		mes "That's correct.";
		171_worker_talk(49,.@old$);
		next;
		mes "[Worker]";
		mes "Yes, I've been hired by Rekenber.";
		171_worker_talk(50,.@cook$);
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";
		mes "What did the person wearing blue clothes over here do here?";
		next;
		cutin "",255;
		mes "[Worker]";
		mes "Me? Well... I'm responsible for making and delivering foods to the people working here.";
		171_worker_talk(51,.@cook$);
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";
		mes "And what about the person next to you?";
		next;
		mes "[Worker]";

> 		mes "I carry waste materials and laboratory equipments.";

Reviewer: ```suggestion
		mes "I carry waste materials and laboratory equipment.";
```

		npctalk "Take a at the condition of the injured.","Kaya Tos#171_cor2",bc_self;
		cloakonnpc "Aas#171_cor2",getcharid(0);
		cloakoffnpc "Aas#171_cor",getcharid(0);
		sleep2 1000;
		next;
		cutin "ep171_as02",0;
		mes "[Aas]";
		mes "Alright, please stay here.";
		next;
		cutin "ep171_kaya01",2;
		mes "[Kaya]";
		mes "Do you know who she was?";
		next;
		cutin "",255;
		mes "[Worker]";
		mes "Yes, she's one of the senior researchers, her name is ^0000CDElyumina.^000000";
		npctalk "There is a nice song in the yard.","Worker#171_rvs_11",bc_self;
		next;
		cutin "ep16_tes01",2;
		mes "[Tes]";
		mes "It was stupid thing to come and attack directly. What are they studying?";
		unittalk getcharid(3),strcharinfo(0) + " : I'd rather say the combatants on both sides were wrong.";
		next;
		cutin "",255;
		mes "[Worker]";
		mes "That's because I'm the only mechanic in the lab. I don't care about anything other than my own machines...";
		next;
		mes "[Worker]";
		mes "Now that I think about it, her eyes were not normal. She looks like a crazy person.";
		npctalk "Yeah. She used to laugh strangely before.","Worker#171_rvs_13",bc_self;
		next;
		mes "[Worker]";

> 		mes "Have you evere seen her room? I was frightened when I saw it the first time. She closed the door quicly, but I saw it.";

Reviewer: ```suggestion
		mes "Have you ever seen her room? I was frightened when I saw it the first time. She closed the door quickly, but I saw it.";
```

	end;
}

//= Finding Patrick Kaufmann
sp_cor,132,144,3	script	Worker#171_rvs_11	4_F_05,{
	if (isbegin_quest(17020) == 1) {
		mes "[Worker]";
		mes "I'll have to split the workforce because I only have so few people remaining.";
		npctalk "I have to split the workforce.","",bc_self;
		next;
		mes "[Worker]";
		mes "We'll have to take turns.";
		npctalk "No, that's not very productive.","Worker#171_rvs_13",bc_self;
		next;
		select("Do you know Patrick Kaufmann?");
		mes "[Worker]";
		mes "Who?";
		npctalk "Pat, who?","Worker#171_rvs_12",bc_self;
		next;
		select("Patrick Kaufmann. Facility Maintenance Officer.");
		mes "[Worker]";
		mes "Patrick Kaufmann? Don't you mean Patrick Young?";
		npctalk "I don't think that's the right name.","Worker#171_rvs_14",bc_self;
		next;
		mes "[Worker]";
		mes "He's the logistic guy.";
		npctalk "Facility Maintenance Department....","Worker#171_rvs_13",bc_self;
		next;
		mes "[Worker]";
		mes "Where's the facility department here?";
		npctalk "Don't you know?","",bc_self;
		next;
		mes "[Worker]";
		mes "I knew before I came here!";
		npctalk "That's not possible.","Worker#171_rvs_14",bc_self;
		next;
		mes "[Worker]";

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "Who? Patrick? I don't know him. I don't know many people, I'm only focused on the field work here.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "That's right, that's right! The guy who looked tall and big. I wanted to be his friend because we came from the same place. I couldn't see him one day. Then I was moved.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "Oh... did you come to pick up the materials? I have arranged it as best as I could, but I cannot classify it because I don't know what it is.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "We collect the garbage from each room and then process it. Well, usually that's what we do for neat people. They don't say much.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "That's not it. I only categorize large materials or technical equipment.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "My only goal is being successful, and I've done all sort of things.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "Don't look at me like that. The poorer you are, the better off your family is, we would be fools to not take advantage of these people. At least that's what I thought back then.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "If he did, there would be a bracelet somewhere on the floor. It might not even be there anymore...";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "See, this is my family now. Why do I have to carry such a heavy load? It's a hundred times easier to give up hope than to be alive elsewhere isn't it?";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
	mes "The wrist band is a bracelet. And that is where the names of the specimens are written.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "Even though we suppressed the enemy from the first annihilation operation, the mutants started crawling out again.";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: ```suggestion
		mes "I wonder if those were strengthened through the experimentation in this lab?";
```

> 		mes "Patrick is in the Facility Maintenance Department? That I didn't know. Nobody here worked as their usual

Reviewer: This line is missing content. Above they are both called heart hunters. Additionally BellareS the S is missing.
